### PR TITLE
add reverse transform doc

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/developer_guide/api_transformer.rst [hunk 1]
```diff
@@ -129,5 +129,70 @@ In the controller, you should access your transformer via
...
+            400
+        );
+    }
+
+Then complete the ``reverseTransform`` method from the ``FooTransformer``, I will
+keep the same parameter.
+
+.. code-block:: php
+
+    public function reverseTansform($fooFacade, $fooEntity = null)
```
Comment (juchi) [inline en/developer_guide/api_transformer.rst]: the FacadeInterface type hint should be provided as in the interface

File: en/developer_guide/api_transformer.rst [hunk 2]
```diff
@@ -129,5 +129,70 @@ In the controller, you should access your transformer via
...
+
+        // Check if the entity is valid
+        if ($this->isValid($foo)) {
+            //Save the entity
+
+            return new Response('', 200);
+        }
+
+        // If the entity is not valid, return the violations
+        return new response(
```
Comment (juchi) [inline en/developer_guide/api_transformer.rst]: Response

**Comment by nicolasThal** [comment]
@juchi correction made

**Comment by juchi** [comment]
:+1:

**Pull request merged**